provider/provider.go

```diff
@@ -264,111 +100,73 @@ func DatabricksProvider() *schema.Provider {
 	return p
 }
 
+func providerSchema() map[string]*schema.Schema {
+	kindMap := map[reflect.Kind]schema.ValueType{
+		reflect.String: schema.TypeString,
+		reflect.Bool:   schema.TypeBool,
+		reflect.Int:    schema.TypeInt,
+		// other values will immediately fail unit tests
+	}
+	ps := map[string]*schema.Schema{}
+	attrs := common.ClientAttributes() // TODO: pass by argument
+	for _, attr := range attrs {
+		fieldSchema := &schema.Schema{
+			Type:     kindMap[attr.Kind],
+			Optional: true,
+		}
+		ps[attr.Name] = fieldSchema
+		if len(attr.EnvVars) > 0 {
+			fieldSchema.DefaultFunc = schema.MultiEnvDefaultFunc(attr.EnvVars, nil)
+		}
+	}
+
+	ps["token"].Sensitive = true
+	ps["azure_client_secret"].Sensitive = true
+
+	azCoordinatesDeprecation := "`%s` is deprecated and would be removed in v0.4.0. Please rewrite provider configuration " +
+		"with `host = data.azurerm_databricks_workspace.example.workspace_url` to achieve the same effect. Please check " +
```

Comment:
remove `data.` ?

Reply:
Data is data source. Probably I'll add another couple of examples into main doc later